packages/react-variant-view/src/component/frequencyTable/MutationTumorTypeFrequencyTable.tsx

```diff
@@ -0,0 +1,117 @@
+// this component comes from signal
```

Comment:
Similar to `FrequencyCell`, we should either move this to `cbioportal-frontend-commons` or the depending implementation into signal.

Reply:
@onursumer This table is not done yet, we'll add more columns when new data is available. So I guess maybe leave it in the `react-variant-view` package would be easier to customize?